lib/model/migrations/20220121-02-purge-deleted-forms.js

```diff
@@ -30,8 +30,8 @@ with redacted_audits as (
     where audits."acteeId" = forms."acteeId"
     and forms."deletedAt" is not null
   ), purge_audits as (
-    insert into audits ("action", "acteeId", "loggedAt")
-    select 'form.purge', "acteeId",  clock_timestamp()
+    insert into audits ("action", "acteeId", "loggedAt", "processed")
+    select 'form.purge', "acteeId", clock_timestamp(), clock_timestamp()
```

Comment:
if this processed flag is getting set here, it also probably needs to get set in Forms.purge()

Reply:
yeah, this `insert into audits` is in that file, too. it's not the normal auditing functionality because it updates the audit log for a bunch of deleted forms at once.